src/github.com/getlantern/peerscanner/group.go

```diff
@@ -39,13 +39,13 @@ func (g *group) deregister(h *host) {
 
 	log.Debugf("Deregistering from %v: %v", g.subdomain, h)
 
+	g.existing = nil
```

Comment:
BTW, this isn't entirely academic because when I saw the 409s on fallbacks, I saw errors in the log indicating that requests to CloudFlare to deregister from rotations were failing with 524 Origin Time-out.  If the request was actually received and processed, and timed out on the response phase, that would explain the 409's (which were caused by the nl.fallbacks.getiantem.org rotation being empty).